software/contrib/menu.py

```diff
@@ -54,7 +54,8 @@
     ["StrangeAttractor",  "contrib.strange_attractor.StrangeAttractor"],
     ["Turing Machine",    "contrib.turing_machine.EuroPiTuringMachine"],
 
-    ["_Calibrate",        "calibrate.Calibrate"]  # this one should always be last!
+    ["_Calibrate",        "calibrate.Calibrate"],  # this one should always be last!
+    ["_Update Firmware"   "update_firmware.UpdateFirmware"]    # this one should always be last!
```

Comment:
[required] Missing comma above. I believe that this is what is failing the tests.